### PR TITLE
Adds new integration [Aohzan/ipx800]

### PR DESCRIPTION
**Pull request by Aohzan**
Hello,

I add the [GCE IPX800V4](https://github.com/Aohzan/ipx800) integration

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -27,6 +27,8 @@
   "andvikt/mega_hacs",
   "Anonym-tsk/homeassistant-climate-xiaomi-remote",
   "Antoni-Czaplicki/vulcan-for-hassio",
+  "Aohzan/ecodevices",
+  "Aohzan/ipx800",
```
Comment (ludeeus) [inline integration:31]: You can only add one pr. PR.

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Review by ludeeus — CHANGES_REQUESTED** [review]
CI does not pass

**Comment by Aohzan** [comment]
Wait for https://github.com/home-assistant/wheels-custom-integrations/pull/413

**Review by ludeeus — CHANGES_REQUESTED** [review]
CI does not pass
https://hacs.xyz/docs/publish/include/#before-submitting

**Comment by Aohzan** [comment]
@ludeeus fixed, thanks

**Comment by ludeeus** [comment]
marking this as draft for now as discussed on discord

**Comment by Aohzan** [comment]
Hi @ludeeus 
last tag https://github.com/Aohzan/ipx800/releases/tag/2.4.0 add the authentication on the API call and a check on the host making the request. Tell me if it's OK or not like this

**Review by ludeeus — CHANGES_REQUESTED** [review]
CI does not pass
https://hacs.xyz/docs/publish/include/#before-submitting

**Comment by Aohzan** [comment]
Fixed

**Pull request merged**